R/period_dataset.R

```diff
@@ -187,8 +187,8 @@ create_period_dataset <- function(dataset,
     }
     if (mode == "subperiod") {
       period_ref[[i]] <- pivot_longer(
-        select(dataset, !!!subject_keys, matches(cols[[i]])),
-        matches(cols[[i]]),
+        select(dataset, !!!subject_keys, matches(paste0(cols[[i]], "\\b", sep = ""))),
```

Comment:
Do we need `sep = ""` with `paste0`? Think `sep` would only be needed with `paste` function.

Reply:
you're absolutely right, I'll remove, then ask a couple others to review then we can close?